frontend/app/component/form/data_entry/candidates_votes/CandidatesVotesForm.test.tsx

```diff
@@ -116,7 +123,9 @@ describe("Test CandidatesVotesForm", () => {
 
       renderForm({ recounted: false });
 
-      const candidate1 = await screen.findByTestId("candidate_votes[0].votes");
+      const candidate1 = await screen.findByRole("textbox", {
+        name: `1 ${candidateName(politicalGroupMockData.candidates[0] as Candidate)}`,
+      });
```

Comment:
Discussed with @oliver3: change into a `map()` to create an array of candidate names, then use that array in the `findbyRole()`s.

Reply:
done in https://github.com/kiesraad/abacus/pull/646/commits/70f72e696d1a60a07ae3ebcce16e46b532748ee7